lib/main.coffee

```diff
@@ -101,18 +101,25 @@ module.exports = Hydrogen =
         unless kernel
             kernel = @kernelManager.getRunningKernelFor language
 
+        message = "No running kernel for language `#{language}` found"
```

Comment:
I think this kind of early optimisation makes the code less readable (please, read this comment just as an opinion).